.github/workflows/main.yml

```diff
@@ -0,0 +1,153 @@
+        run: |
+          cp ../branch_with_scripts/changesummary.py changesummary.py
+          cp ../branch_with_scripts/create_commits.sh create_commits.sh
+        working-directory: ./branch
+
+      - name: Copy discovery artifacts
+        run: |
+          cp ../discovery-artifact-manager/discoveries/*.json googleapiclient/discovery_cache/documents/
+        working-directory: ./branch
+
+      - name: Get changed discovery artifacts
+        run: git diff origin/master --name-only -- '*.json' |
+              sed 's/.*\///' > changed_files
+        working-directory: ./branch
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+
+      - name: Install pandas
+        run: pip3 install pandas
+
+      - name: Install google-api-python-client
+        run: pip3 install -e .
+        working-directory: ./branch
+
+      - name: Patch to run the next step
+        run: sed -i -e 's/if credentials is None/if False/g' googleapiclient/discovery.py
+        working-directory: ./branch
+
+      - name: Update the docs
+        run: python3 describe.py
+        working-directory: ./branch
+
+      - name: Run Change Summary
+        run: python3 changesummary.py
+        working-directory: ./branch
+
+      - name: Commit changes
+        run: ./create_commits.sh
+        working-directory: ./branch
+
+      - name: Push changes
+        run: git push -f --set-upstream origin update-discovery-artifacts
+        working-directory: ./branch
+
+      - name: Prepare summary for PR Body
+        id: pr_body
+        shell: bash
+        run: |
+          output=$(cat temp/allapis.summary)
+          output="${output//'%'/'%25'}"
+          output="${output//$'\n'/'%0A'}"
+          output="${output//$'\r'/'%0D'}"
+          echo "::set-output name=change_summary::$output"
+        working-directory: ./branch
+
+      - name: Create PR
+        uses: actions/github-script@v2.0.0
+        with:
+          github-token: ${{secrets.YOSHI_CODE_BOT_TOKEN}}
+          script: |
+              const {owner, repo} = context.repo
+              const branch = 'update-discovery-artifacts'
+              let pr_body = `${{ steps.pr_body.outputs.change_summary }}`
+              let pr_title = 'chore: Update discovery artifacts'
+              const pull_requests = await github.pulls.list({
+                  owner: owner,
+                  repo: repo,
+                  head: `${owner}:${branch}`,
+                  state: 'open'
+                });
+
+              if (pull_requests.data.length == 1) {
+                prNumber = pull_requests.data[0].number
+                await github.pulls.update({
+                  owner: owner,
+                  repo: repo,
+                  pull_number: prNumber,
+                  title: pr_title,
+                  body: pr_body
+                });
+                console.log('Updated PR')
+              }
+              else {
```

Comment:
nit: this formatting seems a little bit off, might  be worth running the JavaScript through a linter once. You can put it into a `foo.js` file and run:

`npx standard --fix foo.js`

Reply:
Done.